dwave/cloud/api/client.py

```diff
@@ -326,6 +326,13 @@ def _retry_config(backoff_max=None, **kwargs):
 
         return retry
 
+    @classmethod
```

Comment:
Looks like this could be a `staticmethod`, though perhaps it makes sense to keep it consistent with some of the other `classmethod`s around.

Reply:
I still haven't decided if this should actually be a regular method, with access to `self.config`, or not. That's why I kept it similar to the helper methods around. And private, so it's easy to change later.